iam/api/ServiceAccountKeys/ServiceAccountKeysCreate.cs

```diff
@@ -34,8 +34,16 @@ public static ServiceAccountKey CreateKey(string serviceAccountEmail)
             new CreateServiceAccountKeyRequest(),
             "projects/-/serviceAccounts/" + serviceAccountEmail)
             .Execute();
-        Console.WriteLine("Created key: " + key.Name);
+
+        // The PrivateKeyData field contains the base64-encoded service account key
+        // in JSON format.
+        // TODO(Developer): Save the below key (jsonKeyFile) to a secure location.
+        //  You cannot download it later.
+        var valueBytes = System.Convert.FromBase64String(key.PrivateKeyData);
+        jsonKeyFile = Encoding.UTF8.GetString(valueBytes);
```

Comment:
It's better if we are explicit in samples, and I think the jsonKeyFile variable is not declared.

```suggestion
        byte[] valueBytes = System.Convert.FromBase64String(key.PrivateKeyData);
        string jsonKeyContent = Encoding.UTF8.GetString(valueBytes);
```